### PR TITLE
Micu/app switch insteadda check box #303

### PR DESCRIPTION
**Pull request by micuintus**

### CODE REVIEW

File: qml/pages/VenueFilterSettings.qml [hunk 1]
```diff
@@ -76,38 +76,38 @@ BVApp.Page {
...
+            TextSwitch  {
                 id: foodButton
                          //% "Eating out"
                 text: qsTrId("id-gastro")
 
-                onClicked: {
+                onCheckedChanged: {
                     // for SFOS: we need to implement "mutual exclusive", since we currently only have a TextSwitch
                     if (checked) {
-                        shoppingButton.checked = false;
```
Comment (jmastr) [inline qml/pages/VenueFilterSettings.qml]: Diff unnecessary. Added two lines later again.

File: qml/pages/VenueFilterSettings.qml [hunk 2]
```diff
@@ -76,38 +76,38 @@ BVApp.Page {
...
+            TextSwitch {
                 id: shoppingButton
                          //% "Shops"
                 text: qsTrId("id-shops")
 
-                onClicked: {
+                onCheckedChanged: {
                     // for SFOS: we need to implement "mutual exclusive", since we currently only have a TextSwitch
                     if (checked) {
-                        foodButton.checked = false;
```
Comment (jmastr) [inline qml/pages/VenueFilterSettings.qml]: Diff unnecessary. Added two lines later again.

File: qml/pages/VenueFilterSettings.qml (hunk 1 above)
Comment (micuintus) [reply qml/pages/VenueFilterSettings.qml]: Changed

File: qml/pages/VenueFilterSettings.qml (hunk 2 above)
Comment (micuintus) [reply qml/pages/VenueFilterSettings.qml]: Changed

**Pull request merged**